BN Translations: Updated translations upto 20%

abhik-99 opened the pull request:
This PR aims to include the new translations and minor corrections in the previous ones. This bumps the BN translations completion from 12% to 20%.

Ref: Issue #653

mirwasi commented on po/bn.po:
```diff
@@ -2020,6 +2026,14 @@ msgid ""
+"manual.html#emacs), [Vim/Neovim](https://rust-analyzer.github.io/manual."
+"html#vimneovim) এবং ইত্যাদির জন্যে স্বয়ং-সম্পূর্ণতা এবং সরাসরি সংজ্ঞায় খুজার মতন "
+"কার্যকারিতা প্রদান করে। আরেটা আই.ডি.ই-ও ব্যবহার করা যেতে পারে এখানে যার নাম "
+"[RustRover](https://www.jetbrains.com/rust/)|"
```
```suggestion
"[RustRover](https://www.jetbrains.com/rust/)।"
```

mirwasi commented on po/bn.po:
```diff
@@ -2088,95 +2105,125 @@ msgid ""
 "clippy](https://github.com/rust-lang/rust-clippy))."
 msgstr ""
+"[cargo clippy](https://github.com/rust-lang/rust-clippy)-র মতন নানান উপ-আদেশ "
+"প্লাগিন কাজে লাগিয়েও প্রসারিত করা যায়|"
```
```suggestion
"প্লাগিন কাজে লাগিয়েও প্রসারিত করা যায়।"
```

mirwasi commented on po/bn.po:
```diff
@@ -2225,13 +2278,18 @@ msgid ""
 "The embedded playgrounds cannot execute unit tests. Copy-paste the code and "
 "open it in the real Playground to demonstrate unit tests."
 msgstr ""
+"অনুবিদ্ধ প্লেগ্রাউন্ডটি একক পরীক্ষা চালাতে পারবে না। একক পরীক্ষাগুলি প্রদর্শনের জন্য "
```
Love this usage of "অনুবিদ্ধ"!

abhik-99 replied:
Thanks!

mirwasi reviewed (approved):
Thank you very much, this is a substantial contribution! Apart from some typos (which you can fix now or later), I think this is good to go. The most common one is the usage of vertical bar instead of দাঁড়ি (।)

abhik-99 commented on po/bn.po:
```diff
@@ -1263,7 +1258,7 @@ msgid ""
 "এটি Google এর অ্যান্ড্রয়েড টিম দ্বারা তৈরি একটি চার দিনের রাস্টশিখবার কোর্স যা "
 "রাস্ট-এর সম্পূর্ণ স্পেকট্রাম, মৌলিক সিনট্যাক্স থেকে শুরু করে জেনেরিক এবং ত্রুটি "
-"পরিচালনার মতো উন্নত বিষয় শিখতে সাহায্য করতে পারে ।"
+"পরিচালনার মতো উন্নত বিষয় শিখতে সাহায্য করতে পারে|"
```
Yeah, I am not sure on the keyboard shortcut for this one. That's what led to the usage of pipe "|" being confused on my part.

mirwasi commented:
> > Thank you very much, this is a substantial contribution! Apart from some typos (which you can fix now or later), I think this is good to go. The most common one is the usage of vertical bar instead of দাঁড়ি (।)
> 
> Yup, but which unicode should be used? Unicode 0964 or 09F7? I got a bit confused here and started using pipe "|".

This is a very good point. Unicode's [recommendation](https://unicode.org/faq/bengali.html#10) (please scroll down to the second last question) is to use the Devanagari danda (U+0964).

abhik-99 commented:
> > > Thank you very much, this is a substantial contribution! Apart from some typos (which you can fix now or later), I think this is good to go. The most common one is the usage of vertical bar instead of দাঁড়ি (।)
> > 
> > 
> > Yup, but which unicode should be used? Unicode 0964 or 09F7? I got a bit confused here and started using pipe "|".
> 
> This is a very good point. Unicode's [recommendation](https://unicode.org/faq/bengali.html#10) (please scroll down to the second last question) is to use the Devanagari danda (U+0964).
> 
> Btw, "Bengali Currency Numerator Four" (U+ 09F7) is something else and not to be used as dari.

Thanks! I have incorporated the changes discussed and squashed the commits.
